### ISSUE TITLE
'cargo install lsp4spl' not working 

### ISSUE THREAD

**Issue by phfn** [AlecGhost/LSP4SPL#23]
Running `cargo install lsp4spl` results in:
```
   Compiling lsp4spl v0.1.0
error[E0063]: missing field `diagnostic_provider` in initializer of `lsp_types::ServerCapabilities`
  --> /home/phfn/.cargo/registry/src/index.crates.io-6f17d22bba15001f/lsp4spl-0.1.0/src/main.rs:47:9
   |
47 |         ServerCapabilities {
   |         ^^^^^^^^^^^^^^^^^^ missing `diagnostic_provider`

For more information about this error, try `rustc --explain E0063`.
error: could not compile `lsp4spl` (bin "lsp4spl") due to previous error
error: failed to compile `lsp4spl v0.1.0`, intermediate artifacts can be found at `/tmp/cargo-installsLZlRB`

```

**Comment by phfn** [comment]
cloning + `cargo build --release` works perfectly fine

**Comment by AlecGhost** [comment]
Interesting. Apparently `cargo install` pulls in todays patch 0.94.1 of `lsp-types`, which breaks the API of the crate.
Building normally works just fine, because `Cargo.lock` fixes the version to 0.94.0.

Anyway. I released a new patch on crates.io which uses the updated version. Could you please verify, that it works?

**Comment by phfn** [comment]
works fine
